virttest/vt_resmgr/resources/network/tap/tap_network.py

```diff
@@ -0,0 +1,51 @@
+import copy
+import logging
+
+from ...pool import ResourcePool
+from ...pool_selector import PoolSelector
+from .tap_port import TapPort
+
+LOG = logging.getLogger("avocado." + __name__)
+
+
+class LinuxBridgeNetwork(ResourcePool):
+    TYPE = "linux_bridge"
+    _SUPPORTED_RESOURCES = {
+        "port": TapPort,
+    }
+
+    @classmethod
+    def define_config(cls, pool_name, pool_params):
+        config = super().define_config(pool_name, pool_params)
+        config["spec"].update(
+            {
+                "switch": pool_params["switch"],
+                "export": pool_params.get("export"),
+            }
+        )
+        return config
+
+    def customize_pool_config(self, node_name):
+        config = copy.deepcopy(self.config)
+        config["spec"]["switch"] = config["spec"]["switch"][node_name]["ifname"]
+        config["spec"]["export"] = config["spec"]["export"][node_name]["ifname"]
+        return config
+
+    def meet_conditions(self, condition_params):
```

Comment:
the function signature changes, now it's 
def meet_resource_request(self, resource_type, resource_params):
and super().meet_resource_request(...) checks if the resource_type can be supported by the pool itself.